cmd/homi/docker/compose/homi.go

```diff
@@ -166,7 +172,7 @@ func (ist *Homi) init(number int, addresses, nodeKeys []string, genesis, scGenes
 			dockerImageId,
 			useFastHttp,
 			networkId,
-			parentChainId,
+			scnParentChainId,
```

Comment:
Do we need to set ParentChainID for just one SCN to enable subbridge?

Reply:
Yes, we need it for testing service chain functions with a light node configuration.

Reply:
Fixed. PTAL.